src/schema/slack/types/custom/custom_slack_types_test.ts

```diff
@@ -0,0 +1,10 @@
+import { assertEquals } from "../../../../dev_deps.ts";
```

Comment:
Where to put this test file was a little awkward — do we place it near the Custom Slack Types that worry about this or do we place it near the `DefineType` definition and import the Custom Slack Types? I went with here thinking that if additional tests needed to be added to Custom Slack Types, it'd be easier to find.